web-ui/src/components/guides/GuideLink.jsx

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import ListItem from '@mui/material/ListItem';
-import ListItemText from '@mui/material/ListItemText';
+import {Link} from 'react-router-dom';
```

Comment:
@timyates We are using Prettier to format our JavaScript code. The file `web-ui/.prettierrc` provides the configuration. Can your editor automatically run Prettier on JavaScript files when you save them?

Reply:
Cool, thanks for the pointer! I imported the rules and applied them to the files I touched :-) https://github.com/objectcomputing/check-ins/pull/2522/commits/8eb29d18e13ea594ace46978a8b140da1bac5184